extensions/blocks/slideshow/transforms.js

```diff
@@ -37,8 +37,8 @@ const transforms = {
 		{
 			type: 'block',
 			blocks: [ 'core/gallery', 'jetpack/tiled-gallery' ],
-			transform: attributes => {
-				const validImages = getValidImages( attributes );
+			transform: ( { images } ) => {
+				const validImages = getValidImages( images );
```

Comment:
Note how we were passing `attributes` to `getValidImages()` instead of `attributes.images`. This would always return 0 valid images.